styles/youtube/catppuccin.user.css

```diff
@@ -597,6 +597,12 @@
       }
     }
 
+    /* disabled buttons */
+    .yt-spec-button-shape-next--disabled {
+       background-color: darken(@surface0, 5);
```

Comment:
```suggestion
       background-color: fade(@overlay0, 70%);
```
How does something like this look?

Reply:
that looks pretty good

Reply:
Screenshot would help thanks.

Reply:
you.. thought it would look like something else?

Reply:
I haven't tested it, would appreciate a screenshot ("how does xyz look") to make sure.

Reply:
Especially since in the original PR comment you didn't link to the video your screenshots were of, so I have no way of locating a video with such a disabled download button.

Reply:
> Especially since in the original PR comment you didn't link to the video your screenshots were of, so I have no way of locating a video with such a disabled download button.

i was watching a dogpack404 video

Reply:
Cool, but I still can't see that video lmao. A link would have worked, in this case just send a screenshot as asked for the previous 3 times.

Reply:
> Cool, but I still can't see that video lmao. A link would have worked, in this case just send a screenshot as asked for the previous 3 times.

https://www.youtube.com/watch?v=NHFvR0ArXPs

Reply:
> ** in this case just send a screensho**

Reply:
>  **in this case just send a screenshot**

oh i thought you meant something else
i'll use the screenshots the next time some case like this happens

Reply:
> You still haven't sent a screenshot...

![image](https://github.com/user-attachments/assets/1c9ece27-77ed-4863-adb6-6e92e3242032)